components/GymPlan/GymPlan.js

```diff
@@ -3,14 +3,27 @@ import { nanoid } from 'nanoid';
 
 import SectionAndExerciseSet from './Section+ExerciseSet';
 
-import { AddSectionButton, StyledForm, SaveButton } from './GymPlanStyling';
+import {
+   AddSectionButton,
+   StyledForm,
+   SaveButton,
+   CenterButtons,
+} from './GymPlanStyling';
 
 export default function GymPlan() {
    const [sections, setSections] = useState([]);
 
    function addSection(sectionName) {
       setSections([...sections, { name: sectionName, exerciseSets: [] }]);
    }
+   // Delete Section
+   function deleteSection(sectionIndex) {
+      if (sectionIndex >= 0) {
+         const removedSection = sections.splice(sectionIndex, 1);
+         sections - removedSection;
```

Comment:
Not sure if I am right, but I think you shouldn't alter your state array (sections) directly, only via the setter-function (setSections). 

so maybe rewrite this line like so: 

```javascript
const newSections = sections - removedSection;
setSections([...newSections]);
```